### PR TITLE
[stable] Fix opBinary[Right]`CustomFloat op CustomFloat` ambiguity

### PR DESCRIPTION
**Pull request by thewilsonator**
For dlang/dmd#8922

### CODE REVIEW

**Comment by thewilsonator** [comment]
This is for stable so the DMD PR can build. Please review carefully.

File: std/numeric.d [hunk 1]
```diff
@@ -568,16 +568,28 @@ public:
     }
 
     /// ditto
+    // Define an opBinary `CustomFloat op CustomFloat` so that those below
```
Comment (edi33416) [inline std/numeric.d]: Since you are here, can you please also fix the type in `opBinary`s doc?
Line 556, from `relavent` to `relevant`.

Thanks

Comment (thewilsonator) [reply std/numeric.d]: Done.

Comment (edi33416) [reply std/numeric.d]: Nice. Thanks

**Comment by thewilsonator** [comment]
Thanks! I don't have merge rights, could you please do it.

**Comment by RazvanN7** [comment]
This needs to be in master in order to unblock https://github.com/dlang/dmd/pull/8922

**Pull request merged**